### PR TITLE
For #44740, shotgun home fix

### PR DESCRIPTION
**Pull request by jfboismenu**
The framework uses SHOTGUN_HOME through the bundled tk-core. I've not actually included it in the PR to reduce the amount of noise. I will do so once it gets the green light to be merged.

These changes are mostly house cleaning chores.

### CODE REVIEW

File: config.ini.example [hunk 1]
```diff
@@ -54,21 +54,9 @@
 # Linux
 # certificate_folder=~/.shotgun/desktop/config/certificates
 
-# WARNING: This option will write sensitive information to the logs. We
```
Comment (jfboismenu) [inline config.ini.example]: This is gone in the new browser integration.

File: config.ini.example [hunk 2]
```diff
@@ -54,21 +54,9 @@
...
-# If set to 1, low level debugging messages from the server will be logged.
-#
-# low_level_debug=0
-
 # Port to listen for requests coming from the browser. If not specified,
 # defaults to 9000. Note that you will need to contact support at
 # support@shotgunsoftware.com in order to have your site communicate on a custom
 # port.
 #
-# port=9000
```
Comment (jfboismenu) [inline config.ini.example]: This is gone in the new browser integration.

File: config.ini.example [hunk 3]
```diff
@@ -54,21 +54,9 @@
...
-
-# Comma separated list of clients whitelisted to access this server. Use this when
-# you have a local server. If not specified, defaults to *.shotgunstudio.com.
```
Comment (jfboismenu) [inline config.ini.example]: This is gone in the new browser integration.

File: debug.ini.example [hunk 4]
```diff
@@ -1,47 +0,0 @@
-[EnvVars]
```
Comment (jfboismenu) [inline debug.ini.example]: This is gone in the new desktop-internal's bootstrap.py

File: python/shotgun_desktop/startup.py [hunk 5]
```diff
@@ -92,9 +97,11 @@ def __desktop_engine_supports_authentication_module(engine):
 
     :returns: True if the engine supports the authentication module, False otherwise.
     """
-    if engine.version.lower() == 'undefined':
+    # Looking for v should be good enough. No need to complicate things with a regular
+    # expression. Github hashes start with a 0-f character.
```
Comment (jfboismenu) [inline python/shotgun_desktop/startup.py]: Fixes the issue with github hashes being lower than vX.Y.Z.

File: python/shotgun_desktop/startup.py [hunk 6]
```diff
@@ -572,22 +581,6 @@ def __getattr__(self, name):
         # proxied class.
         return getattr(self._app_bootstrap, name)
 
-    def get_app_root(self):
```
Comment (jfboismenu) [inline python/shotgun_desktop/startup.py]: Dead code.

File: python/shotgun_desktop/startup.py (hunk 5 above)
Comment (manneohrstrom) [reply python/shotgun_desktop/startup.py]: Do we use a v prefix for github tag descriptors?? Do we have access to core at this point? Could we use the core util methods for version comparison?

Comment (jfboismenu) [reply python/shotgun_desktop/startup.py]: We do have access to core, but we are not guaranteed which version, hence we can't be guaranteed a tank.util.lower_version check would be available, if that's what you are thinking of. Over time it wouldn't be an issue however.

As for git tag descriptors, they use  the tag as is without any manipulation.

**Comment by jfboismenu** [comment]
This has been merged a while ago.

**Pull request closed without merging**